tests/test_service_account.py

```diff
@@ -103,7 +102,9 @@ def _from_json_keyfile_name_helper(payload, scopes=None,
         try:
             with open(filename, 'w') as file_obj:
                 json.dump(payload, file_obj)
-            return ServiceAccountCredentials.from_json_keyfile_name(
+            # TODO(issue 564): Cleanup Credentials access
```

Comment:
One way to do this without introducing a local field is to just wrap it in parentheses:
```
return (
    service_account.ServiceAccountCredentials
    .from_json_keyfile_name)(
    <arguments>)
```
. Up to you though.